public-images/public-images-explanation/lxd-openstack-images.rst

```diff
@@ -0,0 +1,210 @@
+.. _lxd-openstack-images:
+
+LXD and OpenStack images
+========================
+
+`LXD <https://canonical.com/lxd>`__ is an open-source tool for
+orchestrating virtual machines and system containers. It is image based,
+and provides support for a large number of distributions and
+architectures.
+
+`OpenStack <https://ubuntu.com/openstack>`__ is an
+open-source cloud platform designed to create and manage cloud
+resources. By aggregating physical resources such as distributed
+compute, network, and storage into a pool, OpenStack then allocates
+virtual resources on-demand to users out of this pool. It does not
+handle virtualisation itself, but acts as a wrapper that leverages
+existing virtualisation technologies.
+
+
+What are these images?
+----------------------
+
+Canonical provides cloud image artefacts on
+`cloud-images.ubuntu.com <https://cloud-images.ubuntu.com/>`__ that have
+been customised to run on public clouds, including LXD and OpenStack. To
+learn more about these artefacts and supported architectures, visit our
+Ubuntu cloud image artefacts documentation :doc:`here <../public-images-reference/artefacts>`.
+
+
+How do you access them?
+-----------------------
+
+Go to `cloud-images.ubuntu.com <https://cloud-images.ubuntu.com/>`__ and
+select a release. For the latest LTS release, Ubuntu 24.04 LTS Noble
+Numbat, you would navigate to ``noble > current``. Note that all artefacts are
+architecture specific, in the format
+``<release>-<type>-cloudimg-<architecture>-<artefact>``.
+
+LXD and OpenStack also have `Minimal Cloud
```

Comment:
To maintain consistency with the rest of the documentation set, can we remove the capitalisation and keep it as 'minimal cloud images'?